### PR TITLE
Fix compatibility with latest master.

### LINKED ISSUE #103: Performing a transaction doesn't popup the signer dialog
Performing a simple transaction doesn't pop up the signer in the extension's overlay. I'm not sure when this broke. I think it stopped working after I upgraded to parity 1.8.0. If you open parity, you'll see the signer popups there.

There are some errors that get thrown as the plugin is loaded, maybe they are related?

Watching a filter doesn't appear to work anymore either. There is a continuous stream of warnings when attempting to watch a filter.

Example code to reproduce issue: https://gist.github.com/nteymory/446819e6280cf16078cad516a04236a8
Chrome version: 62.0.3202.62
Parity version: 1.8.0-beta
Parity Extension version: 2.7.3

Sample error output from chrome: [console_output.txt](https://github.com/paritytech/parity-extension/files/1404424/console_output.txt)

### PR DESCRIPTION
**Pull request by tomusdrw**
Also closes #103

### CODE REVIEW

File: web3/secureTransport.js [hunk 1]
```diff
@@ -74,6 +74,7 @@ export function createSecureTransport () {
   });
 
   return {
+    ready: Promise.resolve(null),
```
Comment (ngotchac) [inline web3/secureTransport.js:77]: Where is this used?

Comment (tomusdrw) [reply web3/secureTransport.js:77]: Somewhere inside the new Parity codebase.

Comment (ngotchac) [reply web3/secureTransport.js:77]: Ah ok. But so it's OK if it's always `null`?

Comment (tomusdrw) [reply web3/secureTransport.js:77]: Yes. Couldn't find it now in current Parity codebase, I need to check if it's still necessary (might be needed for 1.7/1.8 compatibility).

Comment (ngotchac) [reply web3/secureTransport.js:77]: Okay, that's fine :)

**Review by ngotchac — APPROVED** [review]

**Pull request merged**